src/arrays.js

```diff
@@ -1,61 +1,76 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index < array.length) {
+    return array[index];
+  } else {
+    return array[(index - array.length)];
+  }
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  return (array.join());
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.map(String) 
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+  return strings.map(function(x) {return x.toUpperCase()})
```

Comment:
you can also use `=>` arrow function notation as your callback functions.

e.g:

```
return string.map(x => x.toUpperCase());
```